hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/workflows/PureChecksContext.java

```diff
@@ -34,11 +32,6 @@ public interface PureChecksContext {
     @NonNull
     TransactionBody body();
 
-    /**
-     * Returns the current {@link Configuration}.
-     *
-     * @return the {@link Configuration}
-     */
     @NonNull
-    Configuration configuration();
+    void executeInnerPureCheck(@NonNull TransactionBody body) throws PreCheckException;
```

Comment:
Javadocs missing.
The name of this method should be more generic IMO. Maybe `dispatchPureCheck()`.

Reply:
Fixed